src/Utils/Utf8Decoder.js

```diff
@@ -1,7 +1 @@
-import {
-    TextDecoder as TextDecoderPolyfill,
-} from 'text-encoding-utf-8';
-
-export const TextDecoder = typeof global.TextDecoder === 'function' ? global.TextDecoder : TextDecoderPolyfill;
-
-export default new TextDecoder('utf-8');
+export default new TextDecoder();
```

Comment:
Why not removing `Utf8Decoder` and replacing it by `TextDecoder` directly where it is used?

Reply:
I did it initially to have a single shared instance of `TextDecoder` but I could instantiate it on `B3DMParser` & co.

Reply:
Yes that's what I figured but I think it would be simpler to instantiate it where it's needed (without having a big impact on perfs nor memory)